src/NuGet/BuildNuGets.csx

```diff
@@ -122,6 +124,8 @@ string[] TestPackageNames = {
 var PreReleaseOnlyPackages = new HashSet<string>
 {
     "Microsoft.CodeAnalysis.Build.Tasks",
+    "Microsoft.CodeAnalysis.CSharp.CodeStyle",
+    "Microsoft.CodeAnalysis.VisualBasic.CodeStyle",
```

Comment:
why are these pre-release only?  When we ship a final release, we'll want to upload these?